config/options.go

```diff
@@ -117,6 +118,14 @@ func MaxTagValueLength(maxTagValueLength int) Option {
 	}
 }
 
+// SampleWithoutDebugFlag can be used to decide whether debug flag will be set or not
+// when calling span.setSamplingPriority to force sample a span.
+func SampleWithoutDebugFlag(sampleWithoutDebugFlag bool) Option {
```

Comment:
I think the name can be more descriptive  maybe `AlwaysSetSampleFlagOnSamplingPriorityOne` (or something better)?

Reply:
What about AlwaysSetDebugFlagOnForcedSampling? I feel it's too long.

Reply:
NoDebugFlagOnForcedSampling

Reply:
> NoDebugFlagOnForcedSampling

What about NonDebugForcedSampling?